### PR TITLE
86cuvanyg/bug-fixes-ticket-page

### PR DESCRIPTION
**Pull request by amaljoeqb**
Changes Made:
- Make the barcode larger in mobile view.
- Fix seat layout in ticket overflowing in mobile view.
- Place the home button to bottom.
- Gender in ticket is capitalized.
- Add booking status in ticket.

### CODE REVIEW

**Review by RJR-003 — APPROVED** [review]

File: src/pages/TicketPage/TicketPage.tsx [hunk 1]
```diff
@@ -276,6 +267,19 @@ export const TicketPage = () => {
...
                 </TicketAccordionWrapper>
+                <Button
+                    variant="contained"
+                    onClick={goHomeHandler}
+                    startIcon={<Home />}
+                    sx={{
+                        margin: '2rem 0',
+                        alignSelf: 'center',
+                        textTransform: 'none',
+                        maxWidth: '200px',
```
Comment (akshay-krishna) [inline src/pages/TicketPage/TicketPage.tsx]: Could you try with rem?

**Review by akshay-krishna — APPROVED** [review]

File: src/pages/TicketPage/TicketPage.tsx (hunk 1 above)
Comment (amaljoeqb) [reply src/pages/TicketPage/TicketPage.tsx]: Done

File: src/pages/TicketPage/TicketPage.tsx [hunk 2]
```diff
@@ -276,6 +267,19 @@ export const TicketPage = () => {
                         </Grid>
                     </AccordionDetails>
                 </TicketAccordionWrapper>
+                <Button
```
Comment (sania-salim) [inline src/pages/TicketPage/TicketPage.tsx]: Suggestion that go home button may be placed in the same line as download button. The two primary buttons appear distracting.

File: src/pages/TicketPage/TicketPage.tsx [hunk 3]
```diff
@@ -276,6 +267,19 @@ export const TicketPage = () => {
                         </Grid>
                     </AccordionDetails>
                 </TicketAccordionWrapper>
```
Comment (sania-salim) [inline src/pages/TicketPage/TicketPage.tsx:265]: May change the color of view seat layout button to grey for uniformity.

File: src/components/Ticket/Ticket.tsx [hunk 4]
```diff
@@ -13,7 +13,7 @@ import { Download } from '@mui/icons-material';
 export const Ticket = ({ data }: { data: ITicket }) => {
```
Comment (sania-salim) [inline src/components/Ticket/Ticket.tsx:15]: Due to ticket minwidth, horizontal scroll appears in mobile screen. May reduce the min-witdth to rectify this issue.

Comment (sania-salim) [inline src/components/Ticket/Ticket.tsx:15]: In desktop  and tab views, the seat layout appears at the top of the screen. May center that.

**Review by sania-salim — CHANGES_REQUESTED** [review]

File: src/pages/TicketPage/TicketPage.tsx (hunk 2 above)
Comment (amaljoeqb) [reply src/pages/TicketPage/TicketPage.tsx]: Done

File: src/pages/TicketPage/TicketPage.tsx (hunk 3 above)
Comment (amaljoeqb) [reply src/pages/TicketPage/TicketPage.tsx:265]: Done

File: src/components/Ticket/Ticket.tsx (hunk 4 above)
Comment (amaljoeqb) [reply src/components/Ticket/Ticket.tsx:15]: done

Comment (amaljoeqb) [reply src/components/Ticket/Ticket.tsx:15]: done

**Pull request merged**